packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -132,6 +133,22 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
     );
   }
 
+  protected setVolumeViewportClippingRange() {
```

Comment:
Could this be initVolumeViewportClippingRange?  I usually think of set as something that takes a parameter as the value to apply.  Or resetVolumeViewportClippingRange or some such name?